src/components/FeeLabelToggle/FeeLabelToggle.stories.js

```diff
@@ -25,11 +25,32 @@ import FeeLabelToggle from './FeeLabelToggle';
 import WithThemeDecorator from '../../../storybook/WithThemeDecorator';
 import CenterViewDecorator from '../../../storybook/CenterViewDecorator';
 
+const reduxData = {
+  rates: {
+    BTC: {
+      ETH: 27.228736, EUR: 53175, GBP: 46132, USD: 63591,
+    },
+    ETH: {
+      ETH: 1, EUR: 2017.2, GBP: 1756.24, USD: 2407.1,
+    },
```

Comment:
You only need `ETH: { }` top level rates key, please remove the others as they are noise.